tests/integration/modules/archive.py

```diff
@@ -141,7 +141,7 @@ def test_tar_unpack(self):
 
         # Test extract archive
         ret = self.run_function('archive.tar', ['-xvf', self.arch], dest=self.dst)
-        self.assertTrue(isinstance(ret, list))
+        self.assertTrue(isinstance(ret, list), str(ret))
```

Comment:
Yeah, I was actually not familiar with the 2nd argument to ``asssertTrue()``, I looked at the unittest2 source to figure it out. Since I wasn't sure, I just wanted to make sure that it wasn't something where ``assertTrue()`` was consuming args using something like  ``*args`` in the function definition and asserting truth on all of them, because if that were the case then coercing it to a string would always result in a value that would evaluate as ``True``.